src/indexer/compose.yaml

```diff
@@ -0,0 +1,75 @@
+---
+services:
+  broker:
+    build:
+      context: '.'
+      dockerfile: 'broker/Dockerfile'
+    depends_on:
+      processor:
+        condition: 'service_started'
+    environment:
+      PROCESSOR_WS_URL: 'ws://processor:3008/ws'
+      RUST_LOG: 'info'
+    image: 'econialabs/emojicoin-broker'
+    ports:
+    - '3009:3009'
+    restart: 'unless-stopped'
+
+  postgres:
+    environment:
+      POSTGRES_USER: 'indexer'
+      POSTGRES_DB: 'indexer'
+      POSTGRES_PASSWORD: 'indexer'
```

Comment:
- [ ] Change this and all affected dependents to `emojicoin`, since AWS requires 8 chars or more